doc/faq.md

```diff
@@ -1,8 +1,8 @@
-Q: Do you know if it is possible to use AWS ECR directly in conjunction with DNAnexus as a way to host docker images? It would be good to skip the step of saving the images and uploading them.
+> Q: Do you know if it is possible to use AWS ECR directly in conjunction with DNAnexus as a way to host docker images? It would be good to skip the step of saving the images and uploading them.
 
 A: We currently only support DockerHub and dx:// URLs for docker images. I have notified our developers about this feature suggestion. The more users requesting a feature, it will get higher priority on our development roadmap so we thank you for your feedback.
```

Comment:
There is this https://github.com/dnanexus/dxWDL/blob/master/doc/ExpertOptions.md#private-registries. I don't know if it sufficient, but you can use other repositories.